rails-task-list/app/controllers/people_controller.rb

```diff
@@ -0,0 +1,23 @@
+class PeopleController < ApplicationController
+  def index
+    @people = Person.all
+  end
+
+  def show
+    id = params[:id]
+    @person = Person.find(id)
+  end
+
+  def tasks
+    id = params[:id]
+    @person = Person.find(id)
+    @tasks = @person.tasks
```

Comment:
Do you need this additional instance variable or can you use `@person.tasks` in your view?